2017/proposals/satija-vidur.md

```diff
@@ -0,0 +1,127 @@
+* Trying the code for various examples and writing down various methods that can be implemented to solve the problem.
+* Start to code the on-paper methods, presumably by creating new classes to extend the existing 1D normalization.
+* Creating rough classes with incomplete attributes to test and try the algorithms. 
+
+### June 12th - June 23th, **End of Phase 1**
+
+* Complete the rough classes developed.
+* Making programs to try the implementation.
+* Fixing bugs.
+
+### June 26 - July 7th, **Begin of Phase 2**
+
+* Finishing the creation of 2D Colormaps.
+* Adding exception handling.
+* Start of 2D Colorbar. Jotting down algorithms.
+
+### July 10th - July 21th, **End of Phase 2**
+
+* Implemention of algorithms and testing the bar.
+* Initiating development of color maps for the color bar.
+
+### July 24th - August 4th, **Begin of Phase 3**
+
+* Finishing colormaps for colorbar, checking compatibility with other classes.
+
+### August 7th - August 18th
+
+* Bug fixing, error handling and documentation of everything.
+* Finishing documentation.
+* Saving rest of the time as buffer for unpredictible scenarios. *
+
+### August 21st - August 25th, **Final Week**
+
+Buffer time.
+
+### August 28th - August 29th, **Submit final work**
+
+## Future works
+
+Future work will include making improvements to the work-flow based on the feedback of this project. This will include working on issues and documentation coupled with working on the release version of matplotlib.
+
+## Development Experience
+
+Contributions to Matplotlib:
+
+1. https://github.com/matplotlib/matplotlib/pull/8375
+2. https://github.com/matplotlib/matplotlib/pull/8373
+
+
+Published games to the iOS App Store:
+
+1. https://itunes.apple.com/us/app/cyber-pen-fight/id917996699?ls=1&mt=8
+2. https://itunes.apple.com/us/app/quick-colour/id952603154?ls=1&mt=8
+3. https://itunes.apple.com/us/app/ball-and-bricks/id904503429?ls=1&mt=8
+
+Note : The apps might not be visible on the AppStore as my license has expired.
+
+Made a website at Kites Egypt.
+Certificate : [Kites Egypt](https://drive.google.com/open?id=0B1Eo5dKU2IT2OUhoZzlWbUwxV2s)
+
+## Other Experiences
+
+Currently I'm exploring Machine Learning and making a text based artificial intellegent machine to respond to my texts. 
+
+Link : https://github.com/vidursatija/voiceai
+
+## Why this project?
+
+* It interests me. Plotting applications amaze me and I always wanted to work on one since I first used a plotting application online. I would also like to explore more in open source and develop a stronger grip on python and Machine Learning in general.
+
+* Matplotlib is an extensively used library for plotting. I've previously worked on matplotlib related to Colormaps. I found the code flexible and I loved working on it and would like to contribute more to the community.
+
+* As I already have a fairly solid idea of what to expect and the workings of matplotlib I believe I can contribute to this project in a substantial manner.
```

Comment:
Can you please update your timeline to reflect this? Currently, it seems really thin on details.